components/calendar/selection.md

```diff
@@ -17,11 +17,14 @@ The user can select one or mode dates depending on the Calendar configuration se
 To control how many dates the user can select, use the `SelectionMode` property. It takes a member of the `Telerik.Blazor.CalendarSelectionMode` enum and can be:
 * `Single`
 * `Multiple`
+* `Range`
 
 You can pre-select a date in Single selection mode by setting the `Value` property of the calendar to the desired date.
 
 To pre-select dates in the Multiple selection mode, use the `SelectedDates` property which is of type `List<DateTime>`.
 
+In `Range` selection mode you can get the start and end dates of a selected range, by the user, through the `RangeStart` and `RangeEnd` parameters of type `DateTime`. You also get events `RangeStartChanged` and `RangeEndChanged`. You can read more about them and see an example in the [Events]({%slug components/calendar/events%}) article.
+
 ## Receive User Selection
```

Comment:
I would expect a third item in the list for the range selection

Reply:
I also expect an example for handling range selection

Reply:
The disabled dates also deserve their own example, so perhaps a total of 4 is in order (single selection, multiple selection, range selection, disabled dates) and they can all have headings so they can be linked

Reply:
Added 4 separate examples for Disabled Dates, Single, Multiple & Range selection. Commit: f29afdc730abf62698b27246678231ccb8302838